Form-Controls/index.html

```diff
@@ -13,15 +13,82 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+
+        <section>
+            <label for="customerName">Customer Name:</label>
+            <input 
+                type="text" 
+                id="customerName" 
+                name="customerName" 
+                required 
+                minlength="2" 
+                aria-describedby="name-desc"
+            >
+            <p id="name-desc">Name must be at least 2 characters.</p>
+        </section>
+
+        <hr>
+
+        <section>
+            <label for="customerEmail">Email Address:</label>
+            <input 
+                type="email" 
+                id="customerEmail" 
+                name="customerEmail" 
+                required
+            >
+        </section>
+        
+        </hr>
```

Comment:
`<hr>` is a [void element](https://developer.mozilla.org/en-US/docs/Glossary/Void_element) so it doesn’t need a closing tag.

```suggestion
        <hr>
```